spec/models/header_lookup_spec.rb

```diff
@@ -2,78 +2,45 @@
-      } ,
-      \"results\": {
-          \"bindings\": [
-          {
-            \"d\": { \"type\": \"uri\" , \"value\": \"http://id.nlm.nih.gov/mesh/D018875\" } ,
-            \"dName\": { \"type\": \"literal\" , \"xml:lang\": \"en\" , \"value\": \"Vocabulary, Controlled\" }
-          }
-        ]
-      }
-    }"
-  end
-  let(:empty_mesh_api_response) do
-    "{
-     \"head\": {
-       \"vars\": [ \"d\" , \"dName\" ]
-       } ,
-       \"results\": {
-           \"bindings\": [
-           ]
-       }
-     }"
-  end
-
   let(:no_pid_mesh_subject) { "Fake Mesh Term: DigitalHub field mesh" }
   let(:no_pid_lcsh_subject) { "Fake Lcsh Term: DigitalHub field lcsh" }
 
-  describe "#pid_lookup_by_field" do
-    it 'returns nil for subject header with no pid' do
-      expect(subject.send(:pid_lookup_by_field, no_pid_lcsh_subject, :lcsh)).to eq(nil)
-    end
-
-    it 'returns nil for subject header with no pid' do
-      expect(subject.send(:pid_lookup_by_field, no_pid_mesh_subject, :mesh)).to eq(nil)
-    end
-  end
-
   let(:lcsh_local_subject) { "Cancer" }
   let(:expected_lcsh_local_subject) { "http://id.loc.gov/authorities/subjects/sh85019492" }
 
-  describe "#lcsh_term_pid_local_lookup" do
-    it "returns expected pid" do
-      expect(subject.lcsh_term_pid_local_lookup(lcsh_local_subject)).to eq(expected_lcsh_local_subject)
-    end
-  end
-
   let(:mesh_local_subject) { "Abrin" }
   let(:expected_mesh_local_subject) { "https://id.nlm.nih.gov/mesh/D000036" }
 
-  describe "#mesh_term_pid_local_lookup" do
-    it "returns expected pid" do
-      expect(subject.mesh_term_pid_local_lookup(mesh_local_subject)).to eq(expected_mesh_local_subject)
-    end
-  end
-
   let(:lcnaf_term) { "Birkan, Kaarin" }
   let(:lcnaf_id) { "http://id.loc.gov/authorities/names/n90699999" }
 
-  describe "#lcnaf_pid_lookup" do
-    it "it returns the correct id for the lcnaf term" do
-      expect(subject.lcnaf_pid_lookup(lcnaf_term)).to eq(lcnaf_id)
+  describe "#pid_lookup_by_field" do
+    context "nil values" do
+      it 'returns nil for subject header with no pid' do
+        expect(subject.send(:pid_lookup_by_field, no_pid_lcsh_subject, :lcsh)).to eq(nil)
+      end
+
+      it 'returns nil for subject header with no pid' do
+        expect(subject.send(:pid_lookup_by_field, no_pid_mesh_subject, :mesh)).to eq(nil)
+      end
+    end
+
+    context "#lcsh_term_pid_local_lookup" do
+      it "returns expected pid" do
+        expect(subject.pid_lookup_by_field(lcsh_local_subject, :lcsh)).to eq(expected_lcsh_local_subject)
+      end
+    end
+
+    context "#mesh_term_pid_local_lookup" do
+      it "returns expected pid" do
+        expect(subject.pid_lookup_by_field(mesh_local_subject, :mesh)).to eq(expected_mesh_local_subject)
+      end
+    end
+
+    context "#lcnaf_pid_lookup" do
+      it "it returns the correct id for the lcnaf term" do
+        expect(subject.pid_lookup_by_field(lcnaf_term, :lcnaf)).to eq(lcnaf_id)
```

Comment:
This is the same kind of test as the 2 above, right? If so parallel naming structure would be clearer.